arkane/commonTest.py

```diff
@@ -50,8 +50,6 @@
 from arkane.input import job_list
 from arkane.statmech import InputError, StatMechJob
 
-################################################################################
```

Comment:
Why did you make these modifications?

Reply:
I think we're over using these #### separators, and it looks cleaner w/o them. I could revert this minor change if you'd like.

Reply:
I'm not really opposed to the change, but it's probably worth having a larger discussion first.  I think we should remove this commit from the PR for the time being.